cli/batch.js

```diff
@@ -116,15 +113,15 @@ const getBatchIssue = async (issues) => {
                 .map(
                     (issue, i) =>
                         `\r\n\r\n<details>
-<summary>${i + 1}. ${issue.title} in ${issue.package} ${issue.version} (${
+<summary>${i + 1}. ${issue.issueData.title} in ${issue.pkgName} ${issue.pkgVersions.join('/')} (${
```

Comment:
One of the changes in the API is that each affected package version of an issue was listed separately in the list of issues returned. Now only a single issue is returned with an array of affected package versions (`version` -> `pkgVersions`).

This will have a slight effect on how these batch GitHub issues are rendered. So now the "summary" fold-out lines can have multiple versions:

<img width="667" alt="image" src="https://user-images.githubusercontent.com/10602/154214255-bd89ca37-9fdd-4453-8f80-e50f87880ff5.png">

However, I don't see that as an issue based on how we work, right?

Reply:
I agree, the change doesn't have any meaningful impact.